validation.js

```diff
@@ -1,5 +1,62 @@
-// const form = document.querySelector('form');
-// const name = document.querySelector('#name');
-// const spanName = document.querySelector('#spanName');
-// const email = document.querySelector('#mail');
-// const spanEmail = document.querySelector('#spanEmail');
+const form = document.querySelector('form');
+const email = document.getElementById('mail');
+const emailError = document.querySelector('#mail + span.error');
+const fullName = document.getElementById('name');
+const nameError = document.querySelector('#name + span.error');
+
+function showError() {
+  if (fullName.validity.valueMissing) {
+    nameError.textContent = 'You need to enter your full name.';
+  } else if (fullName.validity.patternMismatch) {
+    nameError.textContent = 'Entered value needs to be a full name.';
+  } else if (email.validity.valueMissing) {
+    // If the field is empty,
+    // display the following error message.
+    emailError.textContent = 'You need to enter an email address.';
+  } else if (email.validity.typeMismatch) {
+    // If the field doesn't contain an email address,
+    // display the following error message.
+    emailError.textContent = 'Entered value needs to be an email address.';
+  } else if (/[A-Z]/.test(email.value)) {
+    emailError.textContent = 'Entered value has to be in lower case';
+  }
```

Comment:
- [ ] Good job working on this, but as per the requirement, the error is supposed to appear when a user tries to submit the form. Please kindly run the validation check when a user tries to submit the form.